ludwig/utils/misc_utils.py

```diff
@@ -91,6 +92,14 @@ def get_from_registry(key, registry):
         raise ValueError(f"Key {key} not supported, available options: {registry.keys()}")
 
 
+def get_default_from_registry(registry) -> Tuple[str, object]:
+    from ludwig.utils.registry import DEFAULT_KEYS
+
+    default_v = get_from_registry(DEFAULT_KEYS[0], registry)
```

Comment:
Could you add a descriptive comment here? Took me a few moments to understand how this worked.

Reply:
removed - see new logic :)